System/Console/ANSI/Example.hs

```diff
@@ -264,4 +266,121 @@ titleExample = do
     pause
     -- Yup, I'm a new title! - ansi-terminal-ex - 83x70
     ---------------------------------------------------
-    -- Title Demo
\ No newline at end of file
+    -- Title Demo
+
+codesExample :: IO ()
+codesExample = do
```

Comment:
I am not sure for whom this example would be useful. Perhaps you could explain what you had in mind?